src/pyiem/ncei/igra.py

```
@@ -261,16 +262,32 @@ def process_sounding(text: str) -> Sounding:
 
 def process_ytd(filename: str):
     """Process the YTD file on NCEI's webserver."""
-    msg = ""
+
+    def _parser(text: str):
+        """Protect the parser."""
+        try:
+            return process_sounding(text)
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)